src/components/generate/index.js

```diff
@@ -0,0 +1,21 @@
+import Input from '../input/index';
+import Select from '../select/index';
+import Button from '../button/index';
+import { useState } from 'react';
+
+const Generate = () => {
+    const [inputTextVal, setInputTextVal] = useState('');
+    const [validity,setValidity] = useState('15 Minutes');
```

Comment:
don't use strings for options like this, create an object using Object.freeze and use keys in state and for operations, values should be used only for rendering
refer here - [object.freeze example](https://github.com/DemonDaddy22/all-about-reactJS/blob/e8437a8cc4fdab126d5c1b7aa65d3ad493bde12f/src/components/StonePaperScissor/components/ResultView/index.js#L9)

Reply:
Done!